docs/fud/xilinx.md

```diff
@@ -91,6 +90,19 @@ You will also need to configure the stage to point to your installations of [Vit
     xilinx_location: /scratch/opt/Xilinx/Vitis/2020.2
     xrt_location: /opt/xilinx/xrt
 
+The setup scripts can also be invoked manually.
+
+    source <Vitis_install_path>/Vitis/2020.1/settings64.sh
+    source /opt/xilinx/xrt/setup.sh
+
+On some Ubuntu distributions, you must also export LIBRARY\_PATH.
+
+    export LIBRARY_PATH=/usr/lib/x86_64-linux-gnu
```

Comment:
Since this section is about how to use fud, let's move this description down to the "How it Works" section, which is all about exactly what happens under the hood. You can see I have some commands under there showing how to manually invoke the tools.

Also, in Markdown, I recommend putting backticks around stuff like `LIBRARY_PATH`. Then you don't need to use \ for escapes.

Reply:
Should I direct authors to the tutorial repository at the end for more information?

Reply:
Good question. I honestly think that tutorial is sometimes useful, sometimes not—their Makefile infrastructure is so insanely complicated. So maybe we should continue linking to just the parts we think are most relevant/useful rather than to the entire thing by itself.